clients/src/main/java/org/apache/kafka/common/security/plain/PlainSaslServer.java

```diff
@@ -79,7 +79,7 @@ public PlainSaslServer(JaasContext jaasContext) {
         }
         if (tokens.length != 3)
             throw new SaslException("Invalid SASL/PLAIN response: expected 3 tokens, got " + tokens.length);
-        authorizationID = tokens[0];
+        String authorizationIDFromClient = tokens[0];
```

Comment:
Nit: should probably `authorizationIdFromClient` to follow our convention.

Reply:
Done.